aws-observability-terraform/app-modules/README.md

```diff
@@ -3,14 +3,15 @@
 | Name | Version |
 |------|---------|
 | <a name="requirement_terraform"></a> [terraform](#requirement\_terraform) | >= 0.13.0 |
-| <a name="requirement_null"></a> [null](#requirement\_null) | ~> 2.1 |
-| <a name="requirement_sumologic"></a> [sumologic](#requirement\_sumologic) | ~> 2.6.2 |
+| <a name="requirement_null"></a> [null](#requirement\_null) | >= 2.1 |
+| <a name="requirement_sumologic"></a> [sumologic](#requirement\_sumologic) | >= 2.6.2 |
 
```

Comment:
rest api provider?

Reply:
Not using restAPI provider anywhere in AWS Obs Soln.